packages/core/src/App/Components/Elements/LandscapeBlocker/landscape-blocker.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { isDisabledLandscapeBlockerRoute } from '@deriv/shared';
+import './landscape-blocker.scss';
+
+const LandscapeBlocker = () => {
+    const location = useLocation();
+    const pathname = location?.pathname;
+    const is_hidden_landscape_blocker = isDisabledLandscapeBlockerRoute(pathname);
+
+    if (is_hidden_landscape_blocker) return null;
+
+    return (
+        <div id='landscape_blocker' className='landscape-blocker'>
+            <div className='landscape-blocker__icon'>
+                <svg xmlns='http://www.w3.org/2000/svg' width='80' height='62' fill='none'>
```

Comment:
can we put this svg in a file?

Reply:
@henry-deriv sure, you are right, it would be better. I just moved this code from index.html and didn't think about this, nice catch, I'll update my PR